apps/prairielearn/src/question-servers/calculation-worker.js

```diff
@@ -125,10 +125,6 @@ function getLineOnce(rl) {
   });
 }
 
-if (require.main !== module) {
-  throw new Error('This script is designed to be run as the main process');
-}
-
```

Comment:
This was meant as a safely check, but I think it's safe enough to get rid of it 🙃

Reply:
Agreed